docs/en/stack/ml/anomaly-detection/ml-limitations.asciidoc

```diff
@@ -106,89 +142,155 @@ For more information about `missing_field_count`,
 see the {ref}/ml-get-job-stats.html[get {anomaly-job} statistics API].
 
 
-[float]
-== Terms aggregation size affects data analysis
-//See x-pack-elasticsearch/#601
+[discrete]
+=== Security integration
 
-By default, the `terms` aggregation returns the buckets for the top ten terms.
-You can change this default behavior by setting the `size` parameter.
+When the {es} {security-features} are enabled, a {dfeed} stores the roles of the
```

Comment:
Maybe just me, but it isn't clear to me if after updating roles the datafeed runs with the new permissions or the original permissions. ` the datafeed subsequently runs with the new permissions` and `datafeed continues to run with the permissions that were associated with the original roles` appear to contradict each other.